src/network/networkctl.c

```diff
@@ -1633,9 +1644,15 @@ static int link_status_one(
                            TABLE_SET_ALIGN_PERCENT, 100,
                            TABLE_STRING, strna(link),
                            TABLE_EMPTY,
+                           TABLE_STRING, "Link File Dropins:",
+                           TABLE_STRV, link_dropins ?: STRV_MAKE("n/a"),
+                           TABLE_EMPTY,
```

Comment:
I prefer to show the dropins like `systemctl` does. I.e., with some decorations at the beginning of each dropin, in stead of adding the new entry for dropins.

Reply:
I'm not sure how to make this work with the table helpers, systemctl does not use the table helper at all

Reply:
How about to create a strv with `link` and `link_dropins`, whose first element is equivalent to `link`, and the subsequent elements are dropins with some decorations? Then, you can pass the strv with `TABLE_STRV`.